website/source/docs/upgrade/upgrade-specific.html.md

```diff
@@ -25,8 +25,24 @@ images to load, in 0.5.5 it has been changed to a single string. No
 functionality was changed. Even if more than one item was specificed prior to
 0.5.5 only the first item was used.
 
-If you have jobs that use the `load` option first upgrade all nodes to 0.5.5,
-then resubmit those jobs with `load` as a single string.
+To do a zero-downtime deploy with jobs that use the `load` option:
+
+* Upgrade servers to version 0.5.5 or later.
+
+* Deploy new client nodes on the same version as the servers.
+
+* Resubmit jobs with the `load` option fixed and a constraint to only run on
+  version 0.5.5 or later:
+
+```hcl
+    constraint {
+      attribute = "nomad.version"
+      operator  = "version"
+      value     = ">= 0.5.5"
+    }
+```
```

Comment:
@schmichael after we ran with this we ended up realizing the syntax is actually a bit different - here's the working snippet:

```hcl
  constraint {
    attribute = "${attr.nomad.version}"
    operator  = "version"
    value     = ">= 0.5.5"
  }
```